frontends/.next/build-manifest.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
once committed, git does not automatically delete the files when you add them to `.gitignore`. you'll have to git-delete them manually again